sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -56,15 +57,27 @@ export class AssessmentAESTController extends BaseController {
   /**
    * Get all pending and declined requests related to an application which would result
    * a new assessment when that request is approved.
-   * @param applicationId application id.
+   * @param applicationId parent application id.
    * @returns assessment requests or exceptions for a student application.
    */
+  @ApiNotFoundResponse({
+    description: `Current application for provided parent application not found.`,
+  })
   @Get("application/:applicationId/requests")
   async getRequestedAssessmentSummary(
     @Param("applicationId", ParseIntPipe) applicationId: number,
   ): Promise<RequestAssessmentSummaryAPIOutDTO[]> {
+    const currentApplicationId =
+      await this.applicationService.getApplicationIdByParentApplicationId(
+        applicationId,
+      );
+    if (!currentApplicationId) {
+      throw new NotFoundException(
+        `Current application for application ${applicationId} was not found.`,
```

Comment:
Please match the not found error to match the API response. ``Current application for provided parent application ${applicationId} was not found.``